include/secure_services.h

```diff
@@ -77,6 +77,19 @@ int spm_request_read(void *destination, u32_t addr, size_t len);
  */
 int spm_firmware_info(u32_t fw_address, struct fw_info *info);
 
+/** Prevalidate a B1 update
+ *
+ * This is performed by the B0 bootloader.
```

Comment:
Should we state here that S0 is used when being booted from S0, and S1 is used when booting from S0?

Reply:
How about adding a sentence to the param docs for dst_addr:

     * @param[in]  dst_addr  Target location for the upgrade. This will typically
     *                       be the start address of either S0 or S1.

Reply:
Done